### ISSUE TITLE
Include Mozilla Root Store Policy Definition of Technically Constrained

### ISSUE THREAD

**Issue by christopher-henderson** [mozilla/tls-observatory#341]
[The Mozilla Root Store Policy](https://www.mozilla.org/en-US/about/governance/policies/security-group/certs/policy/) contains wording in section 5.3.1 which extends the definition of technical constraint to also mean:

`If the certificate includes the id-kp-emailProtection extended key usage, it MUST include the Name Constraints X.509v3 extension with constraints on rfc822Name, with at least one name in permittedSubtrees, each such name having its ownership validated according to section 3.2.2.4 of the Baseline Requirements.`

We would like the Observatory to return this extended definition of technical constraints alongside the baseline definition within a versioned JSON field which contains Mozilla policy information. E.G. something like

```json
...
"MozillaPolicyV2_5": {
    "IsTechnicallyConstrained": true
}
...
```

**Comment by jcjones** [comment]
(This issue is a dupe of https://github.com/mozilla/tls-observatory/issues/255)

**Comment by jcjones** [comment]
Closed by #342
